docs/named_releases.rst

```diff
@@ -0,0 +1,324 @@
+#######################
+Open edX Named Releases
+#######################
+
+The Open edX community can share knowledge and improvements more easily when most people use the same stable, consistent
+version of the Open edX codebase. To that end, edX is creating a series of "Open edX named releases", which are distinct
+from the daily deployments to edx.org and have a longer release cycle (on the order of several months between each
+release). These releases will be tested both by edX and by the Open edX community, and with each subsequent release we
+will include instructions for how to upgrade from the previous Open edX named release.
+
+Latest Open edX Release
```

Comment:
Would "Supported Open edX Release" provide more information with the same amount of words?